### ISSUE TITLE
Is there anyone who tried pybind11 with Clion for windows 10?

### ISSUE THREAD

**Issue by samuel0929** [pybind/pybind11#2070]
I checked the docs and examples, most of them are focus on Linux, and for windows, VS has been suggested. I've tried a Clion project with mingw64, and it can successfully generate the pyd file(like clionPrj.cp37-win_amd64.pyd), but it can not been imported to Python, always "ImportError: DLL load failed: The specified module could not be found". Any one can show me any experience for that? Thanks a lot!

**Comment by liff-engineer** [comment]
Let's assume your project name is **clionPrj**,and you don't change the default output file name. In your code ,it should be :
```C++
PYBIND11_MODULE(clionPrj,m)
```
Your `.pyd` file name must same as your `PYBIND11_MODULE` name.And use in python like this:
```python
from clionPrj import YourClass,YourFunc
```
If the problem persists, your project maybe depends some dynamic library(dll) cann't find by python.

You can use same C++ code(without pybind11 part) to create a example(exe or dll) project ,and find depends DLLs by [Dependency Walker](https://www.dependencywalker.com/).

Just copy depend DLLs to your `.pyd` directory and try.

**Comment by bstaletic** [comment]
No further discussion happened, so I guess the original question is resolved.

**Comment by Kun-Ming** [comment]
Can you please share the cmake code? I have problem on building projects.

**Comment by liff-engineer** [comment]
> Can you please share the cmake code? I have problem on building projects.

look at [pybind11-examples](https://github.com/liff-engineer/pybind11-examples)

**Comment by YannickJadoul** [comment]
Can't really read much, there. But @Kun-Ming posted his own issue, and already resolved it: #2541. Seemed to have been reasonably simple to fix.

**Comment by syhanjin** [comment]
I also encountered this problem and found what might be the real cause. I am using a conda virtual environment, and if it is used in CMakeList.txt,
```cmake
find_package(Python3 COMPONENTS Interpreter Development)
```
Found is the system Python interpreter, which causes the identified Python version to be inconsistent with the actual Python version used
The solution is to comment out this line and select Python integration in the CMake configuration
